src/main/config/configuration.properties

```diff
@@ -20,3 +20,7 @@ registry = https://plugins.dita-ot.org/
 
 # PDF2 defaults
 org.dita.pdf2.i18n.enabled = true
+
+# Compatibility modes
+# Threat keyref elements that contain only whitespace as empty.
+compatibility.keyref.treat-blank-as-empty=true
```

Comment:
@robander @infotexture do you think the name is descriptive enough? It requires docs too, but as a key name